bigquery/snippets/snippet_test.go

```diff
@@ -108,6 +106,11 @@ func TestAll(t *testing.T) {
 		t.Errorf("want table list %q to contain table %q", got, empty)
 	}
 
+	// Print dataset information
```

Comment:
Not sure this comment is needed? If so, please add a period at the end. Same with "Run query variations."